packages/networks/src/networks/mainnet.ts

```diff
@@ -38,6 +38,1028 @@ export const mainnet: NetworkConfig = {
   description: 'The most popular network',
   isTestNetwork: false,
   teamMultisig: '0xa39b44c4AFfbb56b76a1BF1d19Eb93a5DfC2EBA9',
+  tokens: [
+    {
```

Comment:
I think there is too many here... Let's keep the following: 
WETH, DAI, USDC, WBTC, MATIC, BAT and that;s it for now.

Reply:
also keep `USDT` in list